stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -634,6 +645,7 @@
     ;; must be called directly by the tx-sender or by an allowed contract-caller
     (asserts! (check-caller-allowed)
               (err ERR_STACKING_PERMISSION_DENIED))
+    (print {notification: "revoke-delegate-stx", payload: {stacker: tx-sender}})
```

Comment:
Is there a reason to implement this event differently from the others? See here for how the other 10 or so pox events are implements: https://github.com/stacks-network/stacks-core/blob/6d0effc0d1135546196af05c0067a7fd2275fc63/src/clarity_vm/special.rs#L570

Reply:
I thought there was a reason why is wasn't synthesized. If we can do that, I am happy to change the PR. @zone117x

Reply:
I think it was just never implemented. If you can add it to the synthesized events that would help with integration in the Hiro API as well 👍

Reply:
added to `pox-locking/events.rs`